Investigate/fix test_fdb[arp_request] failures seen on backend T0

Failure seen on 10/26 run

-- stack trace --
```
ansible_adhoc = <function init_host_mgr at 0x7f006837fad0>
ptfadapter = <tests.common.plugins.ptfadapter.ptfadapter.PtfTestAdapter testMethod=runTest>
duthosts = <tests.common.devices.duthosts.DutHosts object at 0x7f00682ad350>
rand_one_dut_hostname = 'str2-7050qx-32s-acs-02'
ptfhost = <tests.common.devices.ptf.PTFHost object at 0x7f00666e8210>
pkt_type = 'arp_request', toggle_all_simulator_ports_to_rand_selected_tor = None
record_mux_status = None
@pytest.mark.bsl
@pytest.mark.parametrize("pkt_type", PKT_TYPES)
def test_fdb(ansible_adhoc, ptfadapter, duthosts, rand_one_dut_hostname, ptfhost, pkt_type, toggle_all_simulator_ports_to_rand_selected_tor, record_mux_status):
# Perform FDB clean up before each test and at the end of the final test
fdb_cleanup(duthosts, rand_one_dut_hostname)
if pkt_type == "cleanup":
return
"""
1. verify fdb forwarding.
2. verify show mac command on DUT for learned mac.
"""
duthost = duthosts[rand_one_dut_hostname]
conf_facts = duthost.config_facts(host=duthost.hostname, source="persistent")['ansible_facts']
# reinitialize data plane due to above changes on PTF interfaces
>       ptfadapter.reinit()
ansible_adhoc = <function init_host_mgr at 0x7f006837fad0>
conf_facts = {'ACL_TABLE': {'DATAACL': {'policy_desc': u'DATAACL', 'ports': [u'Ethernet80', u'Ethernet84', u'Ethernet88', u'Etherne...R_PROFILE|ingress_lossy_profile]'}, '3-4': {'profile': u'[BUFFER_PROFILE|pg_lossless_40000_300m_profile]'}}, ...}, ...}
duthost    = <MultiAsicSonicHost> str2-7050qx-32s-acs-02
duthosts   = <tests.common.devices.duthosts.DutHosts object at 0x7f00682ad350>
pkt_type   = 'arp_request'
ptfadapter = <tests.common.plugins.ptfadapter.ptfadapter.PtfTestAdapter testMethod=runTest>
ptfhost    = <tests.common.devices.ptf.PTFHost object at 0x7f00666e8210>
rand_one_dut_hostname = 'str2-7050qx-32s-acs-02'
record_mux_status = None
toggle_all_simulator_ports_to_rand_selected_tor = None
fdb/test_fdb.py:268:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
common/plugins/ptfadapter/ptfadapter.py:139: in reinit
self._init_ptf_dataplane(self.ptf_ip, self.ptf_nn_port, self.device_num, self.ptf_port_set, ptf_config)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <tests.common.plugins.ptfadapter.ptfadapter.PtfTestAdapter testMethod=runTest>
ptf_ip = '10.64.246.158', ptf_nn_port = 10948, device_num = 0
ptf_port_set = [0, 1, 2, 3, 4, 5, ...], ptf_config = None
def _init_ptf_dataplane(self, ptf_ip, ptf_nn_port, device_num, ptf_port_set, ptf_config=None):
"""
initialize ptf framework and establish connection to ptf_nn_agent
running on PTF host
:param ptf_ip: PTF host IP
:param ptf_nn_port: PTF nanomessage agent port
:param device_num: device number
:param ptf_port_set: PTF ports
:return:
"""
self.ptf_ip = ptf_ip
self.ptf_nn_port = ptf_nn_port
self.device_num = device_num
self.ptf_port_set = ptf_port_set
self.connected = False
ptfutils.default_timeout = self.DEFAULT_PTF_TIMEOUT
ptfutils.default_negative_timeout = self.DEFAULT_PTF_NEG_TIMEOUT
ptf_nn_sock_addr = 'tcp://{}:{}'.format(ptf_ip, ptf_nn_port)
ptf.config.update({
'platform': 'nn',
'device_sockets': [
(device_num, ptf_port_set, ptf_nn_sock_addr)
],
'qlen': self.DEFAULT_PTF_QUEUE_LEN,
'relax': True,
})
if ptf_config is not None:
ptf.config.update(ptf_config)
if not self._check_ptf_nn_agent_availability(ptf_nn_sock_addr):
>           raise PtfAdapterNNConnectionError(ptf_nn_sock_addr)
E           PtfAdapterNNConnectionError: Failed to connect to ptf_nn_agent('tcp://10.64.246.158:10948')
device_num = 0
ptf_config =
```

-- logs --
```
26/10/2021 18:29:48 ptfadapter.reinit                        L0134 INFO   | Restarting ptf_nn_agent
26/10/2021 18:29:48 base._run                                L0063 DEBUG  | /azp/agent/_work/1/s/tests/common/plugins/ptfadapter/ptfadapter.py::reinit#135: [vms18-6] AnsibleModule::command, args=["supervisorctl reread"], kwargs={}
26/10/2021 18:29:49 base._run                                L0082 DEBUG  | /azp/agent/_work/1/s/tests/common/plugins/ptfadapter/ptfadapter.py::reinit#135: [vms18-6] AnsibleModule::command Result => {"stderr_lines": [], "cmd": ["supervisorctl", "reread"], "end": "2021-10-26 18:29:49.130306", "_ansible_no_log": false, "stdout": "No config updates to processes", "changed": true, "rc": 0, "start": "2021-10-26 18:29:48.940287", "stderr": "", "delta": "0:00:00.190019", "invocation": {"module_args": {"warn": true, "executable": null, "_uses_shell": false, "strip_empty_ends": true, "_raw_params": "supervisorctl reread", "removes": null, "argv": null, "creates": null, "chdir": null, "stdin_add_newline": true, "stdin": null}}, "stdout_lines": ["No config updates to processes"], "failed": false}
26/10/2021 18:29:49 base._run                                L0063 DEBUG  | /azp/agent/_work/1/s/tests/common/plugins/ptfadapter/ptfadapter.py::reinit#136: [vms18-6] AnsibleModule::command, args=["supervisorctl update"], kwargs={}
26/10/2021 18:29:49 base._run                                L0082 DEBUG  | /azp/agent/_work/1/s/tests/common/plugins/ptfadapter/ptfadapter.py::reinit#136: [vms18-6] AnsibleModule::command Result => {"stderr_lines": [], "cmd": ["supervisorctl", "update"], "end": "2021-10-26 18:29:49.671715", "_ansible_no_log": false, "stdout": "", "changed": true, "rc": 0, "start": "2021-10-26 18:29:49.475353", "stderr": "", "delta": "0:00:00.196362", "invocation": {"module_args": {"warn": true, "executable": null, "_uses_shell": false, "strip_empty_ends": true, "_raw_params": "supervisorctl update", "removes": null, "argv": null, "creates": null, "chdir": null, "stdin_add_newline": true, "stdin": null}}, "stdout_lines": [], "failed": false}
26/10/2021 18:29:49 base._run                                L0063 DEBUG  | /azp/agent/_work/1/s/tests/common/plugins/ptfadapter/ptfadapter.py::reinit#137: [vms18-6] AnsibleModule::command, args=["supervisorctl restart ptf_nn_agent"], kwargs={}
26/10/2021 18:29:52 base._run                                L0082 DEBUG  | /azp/agent/_work/1/s/tests/common/plugins/ptfadapter/ptfadapter.py::reinit#137: [vms18-6] AnsibleModule::command Result => {"stderr_lines": [], "cmd": ["supervisorctl", "restart", "ptf_nn_agent"], "end": "2021-10-26 18:29:52.281144", "_ansible_no_log": false, "stdout": "ptf_nn_agent: stopped\nptf_nn_agent: started", "changed": true, "rc": 0, "start": "2021-10-26 18:29:50.004212", "stderr": "", "delta": "0:00:02.276932", "invocation": {"module_args": {"warn": true, "executable": null, "_uses_shell": false, "strip_empty_ends": true, "_raw_params": "supervisorctl restart ptf_nn_agent", "removes": null, "argv": null, "creates": null, "chdir": null, "stdin_add_newline": true, "stdin": null}}, "stdout_lines": ["ptf_nn_agent: stopped", "ptf_nn_agent: started"], "failed": false}
26/10/2021 18:29:52 utilities.wait_until                     L0089 DEBUG  | Wait until <lambda> is True, timeout is 1 seconds, checking interval is 0.2, delay is 0 seconds
26/10/2021 18:29:52 utilities.wait_until                     L0098 DEBUG  | Time elapsed: 0.000000 seconds
26/10/2021 18:29:52 utilities.wait_until                     L0112 DEBUG  | <lambda> is False, wait 0 seconds and check again
26/10/2021 18:29:52 utilities.wait_until                     L0098 DEBUG  | Time elapsed: 0.200476 seconds
26/10/2021 18:29:52 utilities.wait_until                     L0112 DEBUG  | <lambda> is False, wait 0 seconds and check again
26/10/2021 18:29:52 utilities.wait_until                     L0098 DEBUG  | Time elapsed: 0.401132 seconds
26/10/2021 18:29:52 utilities.wait_until                     L0112 DEBUG  | <lambda> is False, wait 0 seconds and check again
26/10/2021 18:29:52 utilities.wait_until                     L0098 DEBUG  | Time elapsed: 0.601707 seconds
26/10/2021 18:29:52 utilities.wait_until                     L0112 DEBUG  | <lambda> is False, wait 0 seconds and check again
26/10/2021 18:29:53 utilities.wait_until                     L0098 DEBUG  | Time elapsed: 0.802281 seconds
26/10/2021 18:29:53 utilities.wait_until                     L0112 DEBUG  | <lambda> is False, wait 0 seconds and check again
26/10/2021 18:29:53 utilities.wait_until                     L0117 DEBUG  | <lambda> is still False after 1 seconds, exit with False
26/10/2021 18:29:53 __init__.pytest_runtest_call             L0039 ERROR  | Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/_pytest/python.py", line 1464, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File "/usr/local/lib/python2.7/dist-packages/pluggy/hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pluggy/manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pluggy/manager.py", line 87, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "/usr/local/lib/python2.7/dist-packages/pluggy/callers.py", line 208, in _multicall
    return outcome.get_result()
  File "/usr/local/lib/python2.7/dist-packages/pluggy/callers.py", line 81, in get_result
    _reraise(*ex)  # noqa
  File "/usr/local/lib/python2.7/dist-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/usr/local/lib/python2.7/dist-packages/_pytest/python.py", line 174, in pytest_pyfunc_call
    testfunction(**testargs)
  File "/azp/agent/_work/1/s/tests/fdb/test_fdb.py", line 268, in test_fdb
    ptfadapter.reinit()
  File "/azp/agent/_work/1/s/tests/common/plugins/ptfadapter/ptfadapter.py", line 139, in reinit
    self._init_ptf_dataplane(self.ptf_ip, self.ptf_nn_port, self.device_num, self.ptf_port_set, ptf_config)
  File "/azp/agent/_work/1/s/tests/common/plugins/ptfadapter/ptfadapter.py", line 99, in _init_ptf_dataplane
    raise PtfAdapterNNConnectionError(ptf_nn_sock_addr)
PtfAdapterNNConnectionError: Failed to connect to ptf_nn_agent('tcp://10.64.246.158:10948')
```

--- a/tests/common/plugins/ptfadapter/ptfadapter.py
+++ b/tests/common/plugins/ptfadapter/ptfadapter.py
@@ -59,7 +59,7 @@ class PtfTestAdapter(BaseTest):
         sock = nnpy.Socket(nnpy.AF_SP, nnpy.PAIR)
         sock.connect(socket_addr)
         try:
-            return wait_until(1, 0.2, 0, lambda:sock.get_statistic(self.NN_STAT_CURRENT_CONNECTIONS) == 1)
+            return wait_until(10, 0.2, 0, lambda:sock.get_statistic(self.NN_STAT_CURRENT_CONNECTIONS) == 1)
         finally:
             sock.close()
 

